Hannah-Sten/TeXiFy-IDEA
# Index all LaTeX package inclusions

Fix [TEX-104](https://texify-idea.myjetbrains.com/youtrack/issue/TEX-104)

#### Summary of additions and changes

* Add a file based index which saves for each package file which other packages it directly includes
* Add a cache for all indirect inclusions

#### How to test this pull request

Observe no error anymore from missing import inspection:
```latex
\documentclass{article} 
\usepackage{algpseudocode} 
\begin{algorithmic}
\end{algorithmic} 
```
